mypy/build.py

```diff
@@ -2421,6 +2415,25 @@ def find_module_and_diagnose(manager: BuildManager,
             raise CompileError(["mypy: can't find module '%s'" % id])
 
 
+def find_added_packages(suppressed: List[str],
```

Comment:
Style nit: The name seems to imply that this returns the added packages. Maybe rename to `exists_added_packages`, for example.